urllib3/connection.py

```diff
@@ -162,6 +164,38 @@ def connect(self):
         conn = self._new_conn()
         self._prepare_conn(conn)
 
+    def request_chunked(self, method, url, body=None, headers={}):
```

Comment:
`headers={}` as a kw value is a mutability bug waiting to happen, let's do what we do everywhere else (`headers=None` and set the default right after).